tests/PHPUnit/System/expected/test_ManyVisitorsOneWebsiteTest_sortByProcessedMetric_constantRowsCountShouldKeepEmptyRows__API.getProcessedReport_day.xml

```diff
@@ -402,23 +402,8 @@
 		<sum_visit_length>6484</sum_visit_length>
 		<bounce_count>4</bounce_count>
 		<nb_visits_converted>0</nb_visits_converted>
-		<goals>
-			<row idgoal="1">
-				<nb_conversions>8</nb_conversions>
-				<nb_visits_converted>8</nb_visits_converted>
-				<revenue>40</revenue>
-			</row>
-			<row idgoal="2">
-				<nb_conversions>4</nb_conversions>
-				<nb_visits_converted>4</nb_visits_converted>
-				<revenue>20</revenue>
-			</row>
-		</goals>
```

Comment:
Is it expected we're removing total values for goal specific metrics? Seems like it might be useful to keep these.

Reply:
I don't think those values were summed up before... It's a `+` operation on arrays, so guess they were combined and didn't represent a total value. If we want to keep them guess we would need to implement that in a recursive way?

Reply:
I suppose so... @mattab / @tsteur worth keeping these totals (and calculating them correctly)?

Reply:
Ideally yes. Can we maybe use `$row->sumRow` there which may does that?

Reply:
should now be processed recursively